### ISSUE TITLE
Add Article -> Image => loading never stop

### ISSUE THREAD

**Issue by sparisot38** [WWBN/AVideo#4288]
Hello @DanielnetoDotCom 

First of all, happy new year and many thanks for your great dev. Very powerful tool.

I wanted to post an article and upload a picture. But when I click in the tab Image, the upload area never appears. I see that :
<img width="1363" alt="Capture d’écran 2021-01-01 à 20 04 02" src="https://user-images.githubusercontent.com/42521954/103444846-376d7a80-4c6d-11eb-9ce9-99264e11915d.png">

When I save the article and open it again, it works. 
I tried on SAFARI and CHROME.

Any idea ? 
Many thanks 
All the best,

**Comment by DanielnetoDotCom** [comment]
Not sure, can you reproduce it on demo site?

**Comment by sparisot38** [comment]
Cannot see the option menu "Add Article" in the top menu ??

**Comment by sparisot38** [comment]
Hello Again, 
I don't know if it can help and if it is linked the one I shared in my previous message. But I have another issue.
When I want to add a link in the text field of a video or article and ask to open in a new window, I saved it. I can see it the html source code.It is ok. When I saved it and reopen it. It is not saved. It always open in the current windows. This is strange. It worked in a few days again. 

I see also several errors in the log file. Should I post it here ?
Many thanks for your help.
<img width="1332" alt="Capture d’écran 2021-01-02 à 23 41 36" src="https://user-images.githubusercontent.com/42521954/103467950-87733c80-4d54-11eb-94a7-2d8b581e22e6.png">
